tsconfig.release.json

```diff
@@ -7,7 +7,7 @@
     "outDir": "lib",
     "stripInternal": true,
     "target": "es2017",
-    "typeRoots": ["node_modules/@types"]
+    "typeRoots": ["./node_modules/@types"]
```

Comment:
`./` are added to make all references consistent. There are fields in TS config files where paths without `./` are not allowed (such as `extends`), so adding it is the only way to achieve consistency.